### ISSUE TITLE
Ctrl+Mouse scroll changes the font size

### ISSUE THREAD

**Issue by alabuzhev** [microsoft/terminal#3793]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18363.476]
Windows Terminal version (if applicable): No

Any other software? No
```

# Steps to reproduce

Run [the attached app](https://github.com/microsoft/terminal/files/3908161/CtrlScroll.zip), press and hold Ctrl, start scrolling.

# Expected behavior

The app receives `MOUSE_EVENT_RECORD`s and uses them to control the slider.

# Actual behavior

The app receives `MOUSE_EVENT_RECORD`s and uses them to control the slider, **but at the same time the host changes the font size**, so the intended functionality is effectively unusable.

It does not happen in legacy mode and never happened prior to Windows 10.

There is no way to disable this feature in console settings - "Enable Ctrl key shortcuts" is already off.

**Comment by bgshacklett** [comment]
Another unintended consequence is that non-precision touch-pads with momentum can result in the font size being changed unexpectedly.

Step 1: scroll with the touch-pad
Step 2: press the ctrl key shortly after you're finished scrolling
Step 3: marvel at your newly ginormous or insanely small text

This isn't a concern for many, but in environments with corporate-supplied laptops, which I would expect to be a relatively common deployment scenario for Windows Terminal, it's more likely to have a laptop without a precision touch-pad.

**Comment by DHowett** [comment]
@bgshacklett unfortunately, that's true of so many applications. I wish it weren't 😦 but when I scroll in Firefox or Chrome or... well, pretty much any application and then hold down control I get my eyes melted off

**Comment by DHowett** [comment]
WAIT, CHROME DOESN'T DO IT ANYMORE

IT'S FIXABLE???

**Comment by bgshacklett** [comment]
FWIW, Chrome still does it for me.

**Comment by esterkaufman** [comment]
It is happening with any app like webStorm, Chrome, Edge, and Rider.
Should it be fixed soon?

**Comment by artnikbrothers** [comment]
I happens in "Azure DevOps -> code review" as well. Very annoying issue

**Comment by marcelwgn** [comment]
@alabuzhev I've tried the attached app and the font size does not change for me (Or i misunderstood the issue):

![Slider in Terminal increasing and decreasing value](https://github.com/microsoft/terminal/assets/16122379/8c2b2797-6c5b-4fa5-916b-c8e14ddf2766)

The transparency not increasing the value but rather changing opacity is still present though

**Comment by alabuzhev** [comment]
@marcelwgn this report is about conhost/openconsole, not WT.

**Comment by marcelwgn** [comment]
So what do I need to watch out for when running the attached project?

**Comment by alabuzhev** [comment]
@DHowett thanks, but what about
> everything above applies to Ctrl+Shift+Mouse Scroll as well

?

As far as I understand, #19127 does not address that ("Other Ctrl+Scroll functionality (like transparency adjustments) remains unaffected").

**Comment by DHowett** [comment]
> UPD

We missed this because it was not in the title ;P

**Comment by paulinek13** [comment]
@alabuzhev You're right! It's my bad actually, I've missed you mentioned that too in your issue.
So I think I'll create a follow-up PR with an option to disable it :)

**Comment by alabuzhev** [comment]
Thank you!
